org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WChosenboxListEditor.java

```diff
@@ -622,4 +680,399 @@ public void newRecord(int record_ID) {
 			}
 		}
 	}
+	
+	private class WChosenboxListassistante extends Window implements EventListener<Event> {
+		private static final long serialVersionUID = 1223690858387209211L;
+		private Button bAdd, bRemove, bUp, bDown;
+		private SimpleListModel availableModel = new SimpleListModel();
+		private SimpleListModel selectedModel = new SimpleListModel();
+		private Listbox availableList = new Listbox();
+		private Listbox selectedList = new Listbox();
+		private Hlayout hlayout;
+		private Button bOk = ButtonFactory.createNamedButton(ConfirmPanel.A_OK, false, true);
+		private int refID = 0;
+		private String m_newValue = "";
+
+		public WChosenboxListassistante() {
+			super();
+			refID = MColumn.get(Env.getCtx(), gridTab.getTableName(), gridField.getColumnName()).getAD_Reference_Value_ID();
+			setTitle(gridField.getHeader() + " " + Msg.getMsg(Env.getCtx(), "Assistant"));
+			init();
+			load();
+			setClosable(true);
+			setBorder("normal");
+			setShadow(true);
+			setMaximizable(true);
+			setSizable(true);
+			setHeight("600px");
+			setWidth("700px");
```

Comment:
this is not mobile friendly, use ZKUpdateUtil.setWindowHeightX and ZKUpdateUtil.setWindowWidthX instead

Reply:
to use isUseCSSForWindowSize(), you have to change add the configuration for your window to the default theme's css, it is not auto.

Reply:
Hi,
Ok, should be ok now
Thanks